a3p-integration/proposals/f:fast-usdc/operators.test.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-env node */
+import unknownTest from 'ava';
+
+import '@endo/init/legacy.js'; // axios compat
+
+import { LOCAL_CONFIG, makeVstorageKit } from '@agoric/client-utils';
+import { evalBundles } from '@agoric/synthetic-chain';
+
+// XXX ~copied to not take a dependency on Zoe
+/**
+ * @typedef {object} InvitationDetails
+ * @property {unknown} installation
+ * @property {unknown} instance
+ * @property {InvitationHandle} handle
+ * @property {string} description
+ * @property {Record<string, any>} [customDetails]
+ */
+
+/**
+ * @import {VstorageKit} from '@agoric/client-utils';
+ */
+
+const test = /** @type {import('ava').TestFn<{ vstorageKit: VstorageKit}>} */ (
+  unknownTest
+);
+
+const ADD_OPERATORS_DIR = 'add-operators';
+
+/**
+ * @typedef {import('@agoric/ertp').NatAmount} NatAmount
+ * @typedef {{
+ *   allocations: { Fee: NatAmount, USD_LEMONS: NatAmount },
+ *  }} ReserveAllocations
+ */
+
+test.before(async t => {
+  const vstorageKit = makeVstorageKit({ fetch }, LOCAL_CONFIG);
+
+  t.context = {
+    vstorageKit,
+  };
+});
+
+test.serial('add operators', async t => {
+  const { vstorageKit } = t.context;
+
+  const addr = 'wallet.agoric16yg5fg0c3ff6k3y5djfzggt5uhu49xd98l6260.current';
+
+  {
+    // no invitation before
+    const curr = await vstorageKit.readPublished(addr);
```

Comment:
The name `addr` threw me off; I assumed it started with `agoric1...` and did a double-take when I got to here.

not critical.